### PR TITLE
Update for councils page

### PR DESCRIPTION
**Pull request by wrightmartin**
This PR adds an updated FMS for councils page

TODO

- [ ] Make the form work
- [ ] Hide case studies (if case studies aren't ready for launch)

fixmystreet for councils fixmystreet

### CODE REVIEW

**Comment by wrightmartin** [comment]
#1468

**Comment by wrightmartin** [comment]
@BenJam just checking you've seen this

**Comment by BenJam** [comment]
> @BenJam just checking you've seen this

have now!

**Comment by BenJam** [comment]
could @davea or @dracos help with the content of the forms?

File: templates/web/fixmystreet.com/about/council.html [hunk 1]
```diff
@@ -1,21 +1,321 @@
...
+      <div class="councils-features__grid">
+        <div class="councils-features__feature">
+          <div class="councils-features__thumb councils-features__thumb--excellence"></div>
+          <h3>Digital Excellence</h3>
+          <p>FixMyStreet was co-designed with council insiders. We made it into the most user-centred, accessible approach to non-emergency issue reporting available today. And now it’s the leader in its class.</p>
+        </div>
+        <div class="councils-features__feature">
+          <div class="councils-features__thumb councils-features__thumb--duplication"></div>
+          <h3>Minimise duplication </h3>
+          <p>FixMyStreet’s open and transparent approach to local issue reporting that reduces incidents of repeat and duplicate reports. Residents who go to report an issues that’s already in your database will see the exiting report, along with a progress report showing how near it is to a fix.</p>
```
Comment (dracos) [inline templates/web/fixmystreet.com/about/council.html]: s/exiting/existing/

File: templates/web/fixmystreet.com/about/council.html [hunk 2]
```diff
@@ -1,21 +1,321 @@
...
+        </div>
+        <div class="councils-features__feature">
+          <div class="councils-features__thumb councils-features__thumb--customise"></div>
+          <h3>Stay flexible</h3>
+          <p>FixMyStreet is Open standards compliant. That gives you complete customer flexibility to work with additional contractors at any time. Integrate other services as and when you need them, and provide additional functionalities to your residents.</p>
+        </div>
+        <div class="councils-features__feature">
+          <div class="councils-features__thumb councils-features__thumb--bringyourown"></div>
+          <h3>Bring your own</h3>
+          <p>You've already invested in mapping, asset, customer and case management services. FixMyStreet slots right in with your existing portfolio, so you can continue to use the systems your team knows and trusts.</p>
```
Comment (dracos) [inline templates/web/fixmystreet.com/about/council.html]: Non curly apostrophe

File: templates/web/fixmystreet.com/about/council.html [hunk 3]
```diff
@@ -1,21 +1,321 @@
...
+        </div>
+      </div>
+      <a href="#order" class="button order-now-button">Order now</a>
+    </div>
+  </div>
+  <div class="councils-clients">
+    <div class="councils-clients__grid">
+      <div class="councils-clients__client">
+        <a href="https://fixmystreet.bristol.gov.uk/">
+          <img src="[% version('/cobrands/fixmystreet.com/images/fms-for-councils/bristol-logo.png') %]" alt="" />
```
Comment (dracos) [inline templates/web/fixmystreet.com/about/council.html]: These definitely need alt attributes if they are links.

File: templates/web/fixmystreet.com/about/council.html [hunk 4]
```diff
@@ -1,21 +1,321 @@
...
+  <div class="councils-pricing">
+    <div class="councils-content-wrapper">
+      <h2>Pricing</h2>
+      <div class="councils-pricing__grid">
+        <div class="councils-pricing__main-prices">
+
+          <div class="pricing-table">
+            <div class="pricing-table__item">
+              <h3>Simple fault reporting</h3>
+              <p class="pricing-table__item__price"><strong>&pound;7500 per year</strong></p>
```
Comment (dracos) [inline templates/web/fixmystreet.com/about/council.html]: Commas in the numbers?

File: templates/web/fixmystreet.com/about/council.html [hunk 5]
```diff
@@ -1,21 +1,321 @@
...
+              <p>The award-winning FixMyStreet experience seamlessly integrated with your site. A responsive web application for reporting, viewing and discussing problems - branded to match your site's styles and templates. Use FixMyStreet for Councils to keep your residents up to date and route problems by email to the correct individual or team. The internal dashboard eanbles you to manage performance.</p>
```
Comment (dracos) [inline templates/web/fixmystreet.com/about/council.html]: Non-curly apostrophe

File: templates/web/fixmystreet.com/about/council.html [hunk 6]
```diff
@@ -1,21 +1,321 @@
...
+            </div>
+            <div class="pricing-table__item">
+              <h3>Case&hyphen;management Integration</h3>
+              <p class="pricing-table__item__price"><strong>&pound;15000 per year</strong></p>
+              <p>Our award-winning reporting solution &ndash; your existing processes and systems. We’ll integrate FixMyStreet into your customer, case or asset management system ensuring that there's minimal impact on your organisation. Accept reports, automatically publish updates and reports from other channels via your existing tools, directly onto FixMyStreet</p>
```
Comment (dracos) [inline templates/web/fixmystreet.com/about/council.html]: Non curly apostrophe (on a line with a curly one too :) )

File: templates/web/fixmystreet.com/about/council.html [hunk 7]
```diff
@@ -1,21 +1,321 @@
...
+                <div class="accordian-content">
+                  <p class="extra-pricing__price">&pound;3,500 per year or included as standard within our case management service</p>
+                  <p class="extra-pricing__desc">Show your residents the locations, identifiers and status of assets right on the map — all they have to do is pick the right one. Compatible with any industry standard WFS or WMS service.</p>
+                </div>
+              </div>
+              <div class="accordian-item">
+                <a href="#" class="accordian-switch accordian-switch--closed"><h4>Adopted roads</h4></a>
+                <div class="accordian-content">
+                  <p class="extra-pricing__price">&pound;3,500 per year or included as standard within our case management service.</p>
+                  <p class="extra-pricing__desc">Ensure that you only receive reports for roads that you actually manage. Highlight adopted roads and private estates at the point of report creation, saving valuable time,expense and irritation. Compatible with any industry standard WFS or WMS service.</p>
```
Comment (dracos) [inline templates/web/fixmystreet.com/about/council.html]: Missing space after comma

File: templates/web/fixmystreet.com/about/council.html [hunk 8]
```diff
@@ -1,21 +1,321 @@
...
+    </div>
+    </form>
+  </div>
+  <div class="council-order__screenshot">
+  </div>
+</div>
+<div class="credits">
+  <div class="councils-content-wrapper">
+    <ul>
+      <li>Bristol photo https://www.flickr.com/photos/harshilshah/14891180406</li>
```
Comment (dracos) [inline templates/web/fixmystreet.com/about/council.html]: Should be linked

File: web/cobrands/fixmystreet.com/js.js [hunk 9]
```diff
@@ -27,3 +27,38 @@ $(function(){
...
+}
+
+
+$(function(){
+
+  var allPanels = $('.accordian > .accordian-item .accordian-content').hide();
+  var allSwitches = $('.accordian .accordian-switch');
+  openFirstPanel();
+
+  $('.accordian > .accordian-item a').click(function() {
```
Comment (dracos) [inline web/cobrands/fixmystreet.com/js.js]: Clicking on an already open one, it closes and reopens :)

File: templates/web/fixmystreet.com/about/council.html (hunk 1 above)
Comment (abimysoc) [inline templates/web/fixmystreet.com/about/council.html]: Also s/an issues/an issue/

File: templates/web/fixmystreet.com/about/council.html (hunk 6 above)
Comment (abimysoc) [inline templates/web/fixmystreet.com/about/council.html]: And a missing full stop at the end of the paragraph.

**Comment by BenJam** [comment]
post review: could we temporarily hide the case studies and the 'integrated partners' sections please?

**Comment by BenJam** [comment]
from @zarino, the url bar in the top three images has the FCA demo site in it.

**Comment by zarino** [comment]
Could we also make the form input borders darker? `1px solid #e9e9e9` is super pale – at first glance I couldn't even see the inputs at all.

**Pull request merged**